add new eosc-life post (#499)
usegalaxy-eu/website: _posts/2020-09-08-training-wp9-eosc-life.md

---
site: freiburg

> title: The Euopean Galaxy team has been awarded with two EOSC-life training projects

Reviewer: ```suggestion
title: EOSC-Life has awarded two training projects to the European Galaxy team
```

tags:
- training
- grants
---

Within the 1st [EOSC-Life](https://www.eosc-portal.eu/eosc-life) Training Open Call, two out of four proposals have been awarded to the

> European Galaxy team to develop climate science e-learning material and Mentoring and Training opportunities

Reviewer: ```suggestion
European Galaxy team to develop climate science e-learning material and mentoring and training opportunities
```
